src/pilot/config_idrac.py

```diff
@@ -248,6 +444,25 @@ def config_idrac_settings(drac_client, ip_service_tag, password, node):
     return response['is_reboot_required'], job_id
 
 
+def _get_nic_config(self, resource, class_name,
+                    selection_expression, attr_cls):
+    filter_query = (
+        'select * '
+        'from %(class)s '
+        'where %(selection_expression)s') % {
+        'class': class_name, 'selection_expression': selection_expression}
+    doc = self.client.enumerate(resource, filter_query=filter_query)
```

Comment:
I don't see how the above line could work since there is no client variable defined.